test/quantization/test_quantize_fx.py

```diff
@@ -1744,6 +1744,44 @@ def forward(self, x):
         mc = convert_fx(mp)
         mc(torch.rand(4, 1, 4, 4))
 
+    def test_fp32_sum(self):
+        """
+        Verifies that fp32 sum works correctly if it's before or after
+        quantized layers.
+        """
+        class M1(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv1 = nn.Conv2d(1, 1, 1)
+
+            def forward(self, x):
+                x = self.conv1(x)
+                x = torch.stack([x])
+                x = torch.sum(x)
+                return x
+
+        class M2(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv1 = nn.Conv2d(1, 1, 1)
+                self.conv2 = nn.Conv2d(1, 1, 1)
+
+            def forward(self, x):
+                x = self.conv1(x)
+                x1 = torch.stack([x])
+                x1 = torch.sum(x1, dim=0)
+                x2 = self.conv2(x1)
+                return x2
+
+        for cls in (M1, M2):
+            m = cls().eval()
+            m(torch.rand(4, 1, 4, 4))
+            qconfig_dict = {'': torch.quantization.default_qconfig}
+            mp = prepare_fx(m, qconfig_dict)
+            mp(torch.rand(4, 1, 4, 4))
+            mc = convert_fx(mp)
```

Comment:
In this case, shouldnt we be inserting an observer at the output of sum?  Also for stack, do we assume that all the tensors stacked have the same scale and zero-point?

Reply:
> shouldnt we be inserting an observer at the output of sum?

yes, I believe this is implemented as "at the input of conv" instead of "at the output of sum", even though the result is the same

> for stack, do we assume that all the tensors stacked have the same scale and zero-point?

quantized stack currently uses the qparams of the first input for the output.  That could be definitely be improved.